internal/crossplane/types.go

```diff
@@ -41,6 +43,10 @@ type Instance struct {
 	Labels      *Labels
 }
 
+func (i Instance) Ready() bool {
+	return i.Composition.GetCondition(xrv1.TypeReady).Status == corev1.ConditionTrue
```

Comment:
Can we please rename `Composition` here to `Composite`?
Because an instance is technically a composite, not a composition. The composition is the "building plan" for a composite.